README.docker.md

```diff
@@ -28,12 +28,13 @@ SESSION_SECRET=XXX                          # choose any secret
 PUBLIC_KEY=XXX                              # public key of pre-generated key pair
 PRIVATE_KEY=XXX                             # private key of pre-generated key pair
 PROXYPORT=8080
+NODEPORT=2000                               # DELA node port
 ```
 
-To run the currently released version, go to `docker-compose/` and
-run
+To run the currently released version, run
 
 ```
+export COMPOSE_FILE=docker-compose/docker-compose.yml
```

Comment:
Why use the `export` instead of giving the file with `-f` to `docker compose`?

Reply:
I find it easier not having to repeat the argument, but I can update the README to include both